airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py

```diff
@@ -219,6 +219,13 @@ def transform_snowflake(config: Dict[str, Any]):
             dbt_config["oauth_client_id"] = credentials["client_id"]
             dbt_config["oauth_client_secret"] = credentials["client_secret"]
             dbt_config["token"] = credentials["refresh_token"]
+        elif credentials.get("private_key"):
+            f = open("private_key_path.txt", "w")
+            f.write(credentials["private_key"])
+            f.close()
+            dbt_config["private_key_path"] = "private_key_path.txt"
+            if credentials.get("passphrase"):
+                dbt_config["private_key_passphrase"] = credentials["passphrase"]
```

Comment:
Noting that the authentication options that will be added to the Snowflake Destination will be called "private_key" and "passphrase".  Does this match other destinations?

Reply:
"private_key" - is commonly used in other connectors,  "passphrase" - is just snowflake terminology, will rename it to password